contrib/IECoreUSD/src/IECoreUSD/USDScene.cpp

```diff
@@ -141,6 +141,21 @@ T *reportedCast( const IECore::RunTimeTyped *v, const char *context, const char
 	return nullptr;
 }
 
+static pxr::TfToken g_tagsPrimName( "cortexTags" );
+
+bool isSceneChild( const pxr::UsdPrim &prim )
+{
+	if( !prim.IsDefined() || prim.GetName() == g_tagsPrimName )
```

Comment:
This need for manual filtering of `g_tagsPrimName` implies that other DCCs reading USDs written from Cortex/Gaffer are going to be exposing a "/cortexTags" location to users, correct? That's not ideal... but maybe its still better than whatever the DefaultPrim was doing? I assume the use of DefaultPrim was causing an issue?

If other DCCs would be filtering out DefaultPrim on their own, it might have originally seemed like a good way to hide the data from users...

Reply:
I don't think anybody filters out DefaultPrim - at least as I understand it, that is just a bit of metadata to say which prim to reference if the referencer doesn't specify a prim explicitly. Yes, other DCCs may expose a `/cortexTags` location, but they would previously have been exposing a location anyway. I hope to improve on this in more dedicated sets/tags work at some point, but right now I'm just trying to improve the behaviour relating to which prims _we_ expose as locations. I didn't want to use `prim == GetDefaultPrim()` to skip the tags prim because that is clearly nonsense, so I've just tidied things up enough that I can do it based on the name.